web/src/main/java/it/pagopa/selfcare/dashboard/web/controller/UserV2Controller.java

```diff
@@ -0,0 +1,71 @@
+package it.pagopa.selfcare.dashboard.web.controller;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import it.pagopa.selfcare.commons.base.logging.LogUtils;
+import it.pagopa.selfcare.commons.base.security.SelfCareUser;
+import it.pagopa.selfcare.dashboard.connector.model.institution.InstitutionInfo;
+import it.pagopa.selfcare.dashboard.core.UserV2Service;
+import it.pagopa.selfcare.dashboard.web.InstitutionBaseResource;
+import it.pagopa.selfcare.dashboard.web.model.UpdateUserDto;
+import it.pagopa.selfcare.dashboard.web.model.mapper.InstitutionResourceMapper;
+import it.pagopa.selfcare.dashboard.web.model.mapper.UserMapper;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Collection;
+import java.util.List;
+import java.util.UUID;
+
+@Slf4j
+@RestController
+@Api(tags = "user")
+@RequestMapping(value = "/v2", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
you should do [these actions](https://github.com/pagopa/selfcare-onboarding-backend/pull/211) before introduce /v2 path....